docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/templates/content/home_landing/paragraph--ncids-3-feature-card-row.html.twig

```diff
@@ -1,15 +1,15 @@
 {% if (content.field_row_cards['#items']|length > 0) %}
   {% if content.field_container_heading[0] %}
-    <section class="usa-section usa-section--light cgdp-feature-card-row">
-      <div class="grid-container">     
+    <section class="usa-section usa-section--light cgdp-feature-card-row" data-eddl="feature-card">
```

Comment:
When I said `data-eddl-something` I did not mean that everything should be `data-eddl`. This is like having `<div data="foo">` -- kind of makes it hard to use it for multiple purposes.